templates/iot/examples/infinispan/common/010-Config.yaml

```diff
@@ -335,6 +335,15 @@ data:
                                    org.infinispan.query.indexmanager.InfinispanIndexManager
                                </property>
                            </indexing>
+                          <persistence passivation="false">
+                               <file-store
+                                        fetch-state="true"
+                                        read-only="false"
+                                        purge="false"
+                                        path="../../../../../../data/devices/">
```

Comment:
Can we use absolute path here?

Reply:
I tried that and could not get it to work .. It was always relative to `/opt/jboss/infinispan-server/standalone/data/`

Reply:
@dejanb  i took another aproach with rocksDB and it is solved now :)